app/controllers/projects_controller.rb

```diff
@@ -153,6 +153,22 @@ def archived
     @projects = Project.archived
   end
 
+  def reports
```

Comment:
Still not so sure what is the best approach here, so I'll keep it this way until I have more certainty. Better have repetition than the wrong abstraction. It's the first version, let's see how it evolves.